problems/numberOfDigits.js

```diff
@@ -7,8 +7,10 @@
 
 */
 
-function numberOfDigits() {
-
+function numberOfDigits(n) {
+    return n
+    .toString()
+    .split("")
```

Comment:
Do you need this .split("") here? What would you get if you just chained .length to .toString()?